recipe/build.sh

```diff
@@ -14,38 +14,34 @@ if [[ $(uname) == Darwin ]]; then
   export LDFLAGS="-L$PREFIX/lib -Wl,-rpath,$PREFIX/lib -headerpad_max_install_names"
 fi
 
-autoreconf -vfi
-
-# The --enable-silent-rules is needed because Travis CI dies on the long output from this build.
-./configure --prefix=${PREFIX}\
-            --host=$HOST \
-            --enable-linux-lfs \
-            --enable-silent-rules \
-            --enable-shared \
-            --with-ssl \
-            --with-zlib \
-            --with-jpeg \
-            --disable-netcdf \
-            --disable-fortran
-
-make
-
-# ncgen segfaults on macOS
-if [[ $(uname) != Darwin ]]; then
-if [[ "${CONDA_BUILD_CROSS_COMPILATION}" != "1" ]]; then
-make check
-fi
-fi
-
+# The code uses implicit casting, which newer gfortran versions don't allow
+export FFLAGS="${FFLAGS} -fallow-argument-mismatch"
+
+# Link to settings in repository
+ln -s config/cmake/scripts/CTestScript.cmake CTestScript.cmake
+ln -s config/cmake/scripts/HDF4config.cmake HDF4config.cmake
+
+mkdir build
+cd build
+cmake -G "Unix Makefiles" \
```

Comment:
I think this needs `${CMAKE_ARGS}` (https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/)

Reply:
I did wonder about that. On my system, they seem to get picked up automatically.